### PR TITLE
Update the UI of the generator form

### LINKED ISSUE #1458: Update the UI of the generator
**Actual Behaviour**

Currently UI of App Generator is lacking Top navbar, Menu item and Version deployment info

**Expected Behaviour**

* Add top bar similar to Open Event frontend
* Add menu like Google/Susper on the top right corner (http://susper.com/)
* Add Version deployment info to the bottom like http://staging.loklak.org

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Princu7**
Fixes #1458 

**Changes**:
* Adds top bar to the page
* Make a pop over menu similar to Google/Susper
* Add version deployment to the bottom of the page (similar to loklak)
* Indent code in index.html file

**Screenshot**
screenshot from 2017-08-28 17-25-07

**Test Server**
http://secure-meadow-20680.herokuapp.com

CI Tests will be fixed after #1458 is merged!

@mariobehling @aayusharora @niranjan94 Please review. Thanks! :)

### CODE REVIEW

**Comment by aayusharora** [comment]
Can you check Travis @Princu7

File: src/www/js/form.js [hunk 1]
```diff
@@ -94,6 +95,19 @@ $(document).ready(function () {
 
   });
 
+  $('.custom-menubutton').click(function() {
```
Comment (aayusharora) [inline src/www/js/form.js]: cache this variable

File: src/www/js/form.js [hunk 2]
```diff
@@ -227,6 +241,18 @@ $(document).ready(function () {
 
   });
 
+  function deployStatus() {
+    var apiUrl = "https://api.github.com/repos/fossasia/open-event-webapp/git/refs/heads/development";
+    $.ajax({url: apiUrl, success: function(result){
+      var version = result['object']['sha'];
+      var versionLink = 'https://github.com/fossasia/open-event-webapp/tree/' + version;
+      $('#deploy-link').attr('href', versionLink);
```
Comment (aayusharora) [inline src/www/js/form.js]: cache this too.

**Comment by Princu7** [comment]
@aayusharora Thank you so much for reviewing :smiley: I have made the requested changes! Travis will pass after the PR #1448 is merged :)

**Comment by Princu7** [comment]
@aayusharora Thanks for reviewing the PR. The samples had become in an inconsistent state after some of them were updated to the new attributes. Because of it, the Travis was failing since the webapp expected the events to be in the old attributes. After the PR #1448 which completely migrates the code to the new JSON API model of the open-event server, the Travis build which pass again and samples would auto deployed on each PR merge :)

**Pull request merged**